src/components/Notifications/AccountNotification.jsx

```diff
@@ -0,0 +1,347 @@
+  AccordionPanel,
+  ButtonGroup,
+  Button,
+  Container,
+  Grid,
+  GridItem,
+  Text,
+  Link,
+  // useToast,
+} from '@chakra-ui/react';
+
+const AccountNotification = ({
+  notificationBlock,
+  today,
+  removeEntry,
+  approveAfterTimer,
+  idToRemove,
+  setApproveAfterTimer,
+  declineAfterTimer,
+  setDeclineAfterTimer,
+}) => {
+  // const toast = useToast();
+  const [accounts, setAccounts] = useState(notificationBlock.getNotificationData().accounts);
+  const [disableChildrenButtons, setDisableChildrenButtons] = useState(false);
+
+  const blockDate = notificationBlock.getDate();
+  const diffTime = today - blockDate;
+  const diffDays = Math.floor(diffTime / (1000 * 60 * 60 * 24));
+
+  const [removeNotificationBlock, setRemoveNotificationBlock] = useState(false);
+  const [timeoutId, setTimeoutId] = useState(undefined);
+
+  const acceptAll = async accounts => {
+    const timeId = setTimeout(async () => {
+      await Promise.all(
+        accounts.map(async account => {
+          await account.approveCallback();
+        }),
+      );
+      setRemoveNotificationBlock(true);
+      setApproveAfterTimer(true);
+    }, 5000); // set 5 sec timer for accept all requests
+    setTimeoutId(timeId);
+  };
+
+  const declineAll = async accounts => {
+    const timeId = setTimeout(async () => {
+      await Promise.all(
+        accounts.map(async account => {
+          await account.declineCallback();
+        }),
+      );
+      setRemoveNotificationBlock(true);
+      setDeclineAfterTimer(true);
+    }, 5000); // set 5 sec timer for accept all requests
+    setTimeoutId(timeId);
+  };
+
+  const undoAll = async () => {
+    clearTimeout(timeoutId);
+    setTimeoutId(undefined);
+    setRemoveNotificationBlock(false);
+    setApproveAfterTimer(false);
+    setDeclineAfterTimer(false);
+  };
+
+  useEffect(() => {
+    if (approveAfterTimer && removeNotificationBlock) {
+      removeEntry(notificationBlock.key);
+      // toast({
+      //   title: `Approved.`,
+      //   status: 'success',
+      //   duration: 9000,
+      //   isClosable: true,
+      // });
+    } else if (declineAfterTimer && removeNotificationBlock) {
+      removeEntry(notificationBlock.key);
+      // toast({
+      //   title: `Declined.`,
+      //   status: 'info',
+      //   duration: 9000,
+      //   isClosable: true,
+      // });
+    } else if (approveAfterTimer) {
+      setAccounts(accounts => accounts.filter(account => account.id !== idToRemove));
+      // toast({
+      //   title: `Approved.`,
+      //   status: 'success',
+      //   duration: 9000,
+      //   isClosable: true,
+      // });
+    } else if (declineAfterTimer) {
+      setAccounts(accounts => accounts.filter(account => account.id !== idToRemove));
+      // toast({
+      //   title: `Declined.`,
+      //   status: 'info',
+      //   duration: 9000,
+      //   isClosable: true,
+      // });
+    }
+  }, [approveAfterTimer, declineAfterTimer, removeNotificationBlock]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'idToRemove', 'notificationBlock.key', and 'removeEntry'. Either include them or remove the dependency array. If 'removeEntry' changes too often, find the parent component that defines it and wrap that definition in useCallback.